src/ui/THIRD_PARTY_LICENSES.md

```
@@ -1,19 +1,19 @@
 # Third-Party License Inventory
 
-Generated: 2026-08-05 · Packages audited: **226** · Scope: production (server image)
+Generated: 2026-08-24 · Packages audited: **228** · Scope: production (server image)
 
 > Auto-generated by `pnpm licenses:generate`. Do not edit manually.
 
 ## Contents
 
-- [MIT (202 packages)](#mit-202-packages)
+- [MIT (204 packages)](#mit-204-packages)
 - [Apache-2.0 (6 packages)](#apache-2-0-6-packages)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)